src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1824,6 +1825,61 @@ IF(PLUS_USE_OpenCV_VIDEO)
     )
 ENDIF()
 
+#---------------------------------------------------------------------------
+# InfraredSeekCam
+IF(PLUS_USE_INFRARED_SEEK_CAM)
+  # Check OpenCV 
+  IF(NOT PLUS_USE_OpenCV)
+    MESSAGE("PLUS_USE_INFRARED_SEEK_CAM requires PLUS_USE_OpenCV option enabled, enabling.")
+    SET(PLUS_USE_OpenCV ON CACHE BOOL "Use OpenCV for Thermal Infrared Seek Camera." FORCE)
```

Comment:
Could you copy the generic OpenCV message from PlusBuild option?

Reply:
I don't understand your question.

Reply:
Disregard, not critical.